src/internal/components/drag-handle-wrapper/portal-overlay.tsx

```diff
@@ -16,14 +16,14 @@ export default function PortalOverlay({
   isDisabled,
   children,
 }: {
-  track: HTMLElement | null;
+  track: React.RefObject<HTMLElement | null>;
```

Comment:
We need to pass the `RefObject` instead of the `HTMLElement`.

Why? - During testing the `showButtons=true` flag, I discovered that `track` was all the time `null` on when rendering, no arrow buttons have been shown.
This was because `dragHandleRef` on the parent component was null on the initial render (passed null as `track` property). Once DOM element get assigned to the `dragHandleRef` in the parent component's, it won't trigger a re-render of `PortalOverlay` and it stayed null.